### PR TITLE
Updating events/resources page content

### PR DESCRIPTION
**Pull request by kategarklavs**
Just updated this page to reflect the latest content changes.

### CODE REVIEW

**Comment by kategarklavs** [comment]
@femmebot & @line47 Howdy! When one of you gets a chance, would you be able to review this PR? (These are more substantial content changes - otherwise, I would merge this in myself 😌 )

File: pages/events.md [hunk 1]
```diff
@@ -46,51 +44,65 @@ Ask all your application-related questions during our Q&A webinars. Can’t make
...
+
+[SBIR/STTR Funding Agreement Certification (PDF)](https://github.com/18F/nsf-sbir/blob/dev/files/SBIR_STTR_Funding_Agreement.pdf)
+Prospective SBIR/STTR awardees need to complete and submit this form so that the NSF can determine whether they’re eligible for an award. The NSF will use this certification to determine your compliance during the funding term.
+
+[Certificate of Current Cost or Pricing Data (Word doc)](https://github.com/18F/nsf-sbir/blob/dev/files/Certificate_of_Current_Cost.docx)
+The NSF will use this form to certify that your cost and pricing data are accurate and that you’re compliant throughout your funding term.
 
-### [SBIR/STTR Funding Agreement Certification (PDF)](/files/SBIR_STTR_Funding_Agreement.pdf) 
-SBIR/STTR prospective grantees will be notified by NSF to provide this form to determine whether the business is eligible for a Small Business Innovation Research (SBIR) Program award. Certification will be used to ensure continued compliance during the life of the funding agreement.
+[SBIR/STTR Phase I Report Cover Page (PDF)](https://github.com/18F/nsf-sbir/blob/dev/files/SBIR-STTR_Revised_Report_Cover.pdf)
```
Comment (line47) [inline pages/events.md]: I think we should move the PDF, DOC, XLS files under the assets folder. Let me know if you'd like some help with that.

File: pages/events.md [hunk 2]
```diff
@@ -46,51 +44,65 @@ Ask all your application-related questions during our Q&A webinars. Can’t make
...
-### [Certificate of Current Cost or Pricing Data (Word doc)](/files/Certificate_of_Current_Cost.docx)
-Certification that cost and pricing data are accurate; used to ensure continued compliance during the life of the funding agreement.
+## For Phase II applicants and awardees
 
-### [SBIR/STTR Phase I Report Cover Page (PDF) ](/files/SBIR-STTR_Revised_Report_Cover.pdf)
-All SBIR/STTR Phase I and Phase II awardees must complete this certification at all times set forth in the funding agreement. This includes checking all of the boxes and having an authorized officer of the awardee sign and date the certification each time it is requested.
+### Forms to apply for and manage Phase II funding
 
-## Forms for Phase II application and management
+[Company Commercialization History (XLS)](https://github.com/18F/nsf-sbir/blob/dev/files/CommercialHistoryTemplate.xls)
```
Comment (line47) [inline pages/events.md]: Update the files paths to something like for all of the `files` references.
`{{ site.baseurl }}/assets/grantees/files/CommercialHistoryTemplate.xls`

File: pages/events.md (hunk 1 above)
Comment (kategarklavs) [inline pages/events.md]: Oh, that would be awesome! Happy to meet later today, if you like 👍

Comment (line47) [inline pages/events.md]: It looks like I'll need to move these locally and then add to this PR, OK @kategarklavs

Comment (kategarklavs) [inline pages/events.md]: Ah, OK @line47 🎉

**Pull request merged**